src/main/kotlin/no/fdk/concept_catalog/service/SkosApNoModelService.kt

```diff
@@ -173,8 +176,12 @@ class SkosApNoModelService(
 
     private fun addScopeToDefinition(definitionBuilder: DefinitionBuilder, concept: Begrep) {
         concept.omfang
-            ?.takeIf { it.tekst?.isNotBlank() == true || it.uri?.isNotBlank() == true }
-            ?.let { definitionBuilder.scopeBuilder().label(it.tekst, NB).seeAlso(it.uri).build() }
+            ?.takeIf { it.tekst?.isNotBlank() == true || it.uri?.isValidURI() == true }
+            ?.let { source ->
+                val scopeBuilder = definitionBuilder.scopeBuilder()
+                if (!source.tekst.isNullOrBlank()) scopeBuilder.label(source.tekst, NB)
+                if (source.uri.isValidURI()) scopeBuilder.seeAlso(source.uri).build()
```

Comment:
Er dette riktig, det skal være ulikt `buildSource`?

Altså ikke
```suggestion
                if (source.uri.isValidURI()) scopeBuilder.seeAlso(source.uri)
                scopeBuilder.build()
```

Reply:
Det har du nok helt rett i! Fikset.